backend/src/models/Application.ts

```diff
@@ -1,14 +1,29 @@
 import { InferSchemaType, Schema, model } from "mongoose";
 
+/* eslint-disable no-unused-vars */
+export enum Status {
+  APPLIED = "APPLIED",
+  OA = "OA",
+  PHONE = "PHONE",
+  FINAL = "FINAL",
+  OFFER = "OFFER",
+  REJECTED = "REJECTED",
+}
+
```

Comment:
Can you enable `no-unused-vars` after the enum, and also forgot to comment but could you make the enum `Applied = "APPLIED"` to follow the style in the [typescript documentation](https://www.typescriptlang.org/docs/handbook/enums.html).